### ISSUE TITLE
Add a date field

### ISSUE THREAD

**Issue by mrvdb** [galenmaly/lighterpack#49]
I find that for lists I miss a date field. Ideally this would be per list - per item, but one date per item would already be a big help.

Example usage scenarios:
- main gear list: the date when item was acquired 
- in trip list (longer trips): when to plan pick up

**Comment by johnabela** [comment]
I keep track of when I purchase my gear so I can keep track of how many days I have used it, but this is not the type of thing I would care to have displayed on any of my /r/ listings.

**Comment by mrvdb** [comment]
@johnabela I agree, assuming you mean not having the field displayed on a _**shared**_  list
